src/templates/slideshow/slideshow.js

```diff
@@ -0,0 +1,116 @@
+import { React, useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+import './slideshow.scss';
+
+function Slideshow({ content }) {
+  const { images, transitions, animations, logo } = content;
+  const logoClasses = `logo ${logo.position} ${logo.size}`;
+  const [index, setIndex] = useState(0);
+  const [Slide, setSlide] = useState();
+  const timeoutRef = useRef(null);
+  const classes = `image ${transitions} ${animations}`;
+
+  // A random function to simplify the code where random is used
+  function random(multiplier) {
+    return Math.floor(Math.random() * multiplier);
+  }
+
+  // Creates the animation using keyframes from styled components
+  function createAnimation(grow, transform) {
```

Comment:
Functions should a
/** */ comment with parameter descriptions instead of line-comment.